migrations/20200824194331-patient-gender.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.bulkUpdate(
+          'patients',
+          { gender: 'F' },
+          {
+            gender: {
+              [Sequelize.Op.notIn]: ['F', 'Female', 'Male', 'M'],
+            },
+          },
+          { transaction: t }
+        ),
+      ])
+        .then(
+          queryInterface.bulkUpdate(
+            'patients',
+            { gender: 'F' },
+            { gender: 'Female' },
+            { transaction: t }
+          ),
+          queryInterface.bulkUpdate(
+            'patients',
+            { gender: 'M' },
+            { gender: 'Male' },
+            { transacton: t }
+          )
+        )
+        .then(() =>
+          queryInterface.changeColumn(
+            'patients',
+            'gender',
+            {
+              type: Sequelize.DataTypes.ENUM('M', 'F'),
+            },
+            {
+              transaction: t,
+            }
+          )
+        );
+    });
+  },
+
+  down: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.changeColumn(
+          'patients',
+          'gender',
+          {
+            type: Sequelize.DataTypes.STRING,
+          },
+          {
+            transaction: t,
+          }
+        ),
+      ])
+        .then(
+          queryInterface.sequelize.query('DROP TYPE enum_patients_gender;', {
+            transaction: t,
+          })
+        )
+        .then(
+          queryInterface.bulkUpdate(
+            'patients',
+            { gender: 'Male' },
+            { gender: 'M' },
+            { transaction: t }
+          ),
+          queryInterface.bulkUpdate(
+            'patients',
+            { gender: 'Female' },
+            { gender: 'F' },
+            { transaction: t }
+          )
```

Comment:
- only use `Promise.all` when relevant
- `.then()` takes a callback as an argument, so it's necessary to wrap our db call in a callback; i found [this was a great resource](https://javascript.info/promise-chaining) if you want to learn more about promise chaining

```suggestion
      return queryInterface
        .changeColumn(
          'patients',
          'gender',
          {
            type: Sequelize.DataTypes.STRING,
          },
          {
            transaction: t,
          }
        )
        .then(() =>
          queryInterface.sequelize.query('DROP TYPE enum_patients_gender;', {
            transaction: t,
          })
        )
        .then(() =>
          Promise.all([
            queryInterface.bulkUpdate(
              'patients',
              { gender: 'Male' },
              { gender: 'M' },
              { transaction: t }
            ),
            queryInterface.bulkUpdate(
              'patients',
              { gender: 'Female' },
              { gender: 'F' },
              { transaction: t }
            ),
          ])
```